source/unified-test-format/tests/valid-pass/entity-runCursorCommand.yml

```diff
@@ -0,0 +1,115 @@
+          batchSize: 2
+          command: { find: *collection, filter: {}, batchSize: 2 }
+        expectResult: *documents
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collection
+                filter: {}
+                batchSize: 2
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: find
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: long }
+                collection: *collection
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: getMore
+          - commandStartedEvent:
+              command:
+                getMore: { $$type: long }
+                collection: *collection
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: getMore
+
+  - description: createRunCursorCommand creates a cursor and stores it as an entity that can be iterated one document at a time
+    operations:
+      - name: createRunCursorCommand
+        object: *db
+        arguments:
+          commandName: find
+          batchSize: 2
+          command: { find: *collection, filter: {}, batchSize: 2 }
+        saveResultAsEntity: &myRunCommandCursor myRunCommandCursor
+      - name: iterateUntilDocumentOrError
+        object: *myRunCommandCursor
+        expectResult: { _id: 1, x: 11 }
+      - name: iterateUntilDocumentOrError
+        object: *myRunCommandCursor
+        expectResult: { _id: 2, x: 22 }
+      - name: iterateUntilDocumentOrError
+        object: *myRunCommandCursor
+        expectResult: { _id: 3, x: 33 }
+      - name: iterateUntilDocumentOrError
+        object: *myRunCommandCursor
+        expectResult: { _id: 4, x: 44 }
+      - name: iterateUntilDocumentOrError
+        object: *myRunCommandCursor
+        expectResult: { _id: 5, x: 55 }
+
+  - description: createRunCursorCommand's cursor can be closed and will perform a killCursors operation
+    operations:
+      - name: createRunCursorCommand
+        object: *db
+        arguments:
+          commandName: find
+          batchSize: 2
+          command: { find: *collection, filter: {}, batchSize: 2 }
+        saveResultAsEntity: myRunCommandCursor
+      - name: iterateUntilDocumentOrError
+        object: *myRunCommandCursor
+        expectResult: { _id: 1, x: 11 }
+      - name: close
+        object: *myRunCommandCursor
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collection
+                filter: {}
+                batchSize: 2
+                $db: *db
+                lsid: { id: { $$type: binData } }
+              commandName: find
+          - commandStartedEvent:
+              command:
+                killCursors: *collection
+                cursors: [{ $$type: long }]
```

Comment:
`entity-find-cursor.yml` uses `cursors: { $$type: array }`

Reply:
fixed here and in run-command tests